src/get-started/quickstart.md

```diff
@@ -53,6 +53,20 @@ MinimalWordCount.java	WordCount.java
 
 For a detailed introduction to the Beam concepts used in these examples, see the [WordCount Example Walkthrough]({{ site.baseurl }}/get-started/wordcount-example). Here, we'll just focus on executing `WordCount.java`.
 
+## Runner-Specific Dependencies
```

Comment:
This is already done in `examples`; I think I already saw a PR for archetypes too. So, I'd go with the assumption this will be done very shortly. Sure, we'd need a release, but there are other rough edges too -- perhaps we can point to nightlies at the top of the page.